### ISSUE TITLE
`install.sh` does not detect failed installation for write-protected bindir

### ISSUE THREAD

**Issue by mattfredericksen** [dotenv-linter/dotenv-linter#441]
I noticed that when using the `--bindir` flag with a write-protected directory, `install.sh` does not detect that the installation failed.
```
dotenv-linter installer: The latest version will be installed.
install: cannot create regular file '/usr/local/bin/dotenv-linter': Permission denied
sh: 100: /usr/local/bin/dotenv-linter: not found
dotenv-linter installer: Successfully installed  to /usr/local/bin/dotenv-linter
```

I believe this could be fixed by updating 
https://github.com/dotenv-linter/dotenv-linter/blob/017f954640fdc3e8c7ee448c9ec353ec33803d08/install.sh#L97
to
```sh
install "${_temp_dir}/${_file_name}" "${BINDIR}/" || err "Failed to install"
```
Or something similar to that.

**Comment by mgrachev** [comment]
@mattfredericksen 👋 Thank you for your feedback 👍

We will fix that problem 👌
